start-mycroft.sh

```diff
@@ -1,218 +1,218 @@
-    echo "Starting $1"
-    python3 ${_script} $_params
-}
-
-function launch-background() {
-    if ($first_time) ; then
-        echo "Initializing..."
-        ${DIR}/scripts/prepare-msm.sh
-        source-venv
-        first_time=false
-    fi
-
-    name-to-script-path ${1}
-
-    # Check if already running
-    if [[ $( ps aux ) == *${_script}* ]] ; then
-        echo "Restarting: ${1}"
-        source stop-mycroft.sh ${1}
-    else
-        echo "Starting background service $1"
-    fi
-
-    # Security warning/reminder for the user
-    if [[ "${1}" == "bus" ]] ; then
-        echo "CAUTION: The Mycroft bus is an open websocket with no built-in security"
-        echo "         measures.  You are responsible for protecting the local port"
-        echo "         8181 with a firewall as appropriate."
-    fi
-
-    # Launch process in background, sending log to scripts/log/mycroft-*.log
-    python3 ${_script} $_params >> ${scripts_dir}/logs/mycroft-${1}.log 2>&1 &
-}
-
-function launch-all() {
-    echo "Starting all mycroft-core services"
-    launch-background bus
-    launch-background skills
-    launch-background audio
-    launch-background voice
-
-    # Determine platform type
-    if [[ -r /etc/mycroft/mycroft.conf ]] ; then
-        mycroft_platform=$( jq -r ".enclosure.platform" < /etc/mycroft/mycroft.conf )
-        if [[ $mycroft_platform = "mycroft_mark_1" ]] ; then
-            # running on a Mark 1, start enclosure service
-            launch-background enclosure
-        fi
-    fi
-}
-
-function check-dependencies() {
-  if [ ! -f .installed ] || ! md5sum -c &> /dev/null < .installed ; then
-    echo "Please update dependencies by running ./dev_setup.sh again."
-    if command -v notify-send >/dev/null ; then
-      notify-send "Mycroft Dependencies Outdated" "Run ./dev_setup.sh again"
-    fi
-  fi
-}
-
-_opt=$1
-shift
-_params=$@
-
-check-dependencies
-
-case ${_opt} in
-    "all")
-        launch-all
-        ;;
-
-    "bus")
-        launch-background ${_opt}
-        ;;
-    "audio")
-        launch-background ${_opt}
-        ;;
-    "skills")
-        launch-background ${_opt}
-        ;;
-    "voice")
-        launch-background ${_opt}
-        ;;
-
-    "debug")
-        launch-all
-        launch-process cli
-        ;;
-
-    "cli")
-        launch-process ${_opt}
-        ;;
-    "wifi")
-        launch-background ${_opt}
-        ;;
-    "unittest")
-        source-venv
-        pytest test/unittests/ --cov=mycroft "$@"
-        ;;
-    "skillstest")
-        source-venv
-        pytest test/integrationtests/skills/discover_tests.py "$@"
-        ;;
-    "audiotest")
-        source-venv
-        python3 -m mycroft.util.audio_test "${@:1}"
-        ;;
-    "audioaccuracytest")
-        launch-process ${_opt}
-        ;;
-    "sdkdoc")
-        launch-process ${_opt}
-        ;;
-    "enclosure")
-        launch-background ${_opt}
-        ;;
-
-    *)
-        help
-        ;;
-esac
+#!/usr/bin/env bash

```

Comment:
Can you change the line endings back to unix (or setup git to do it automatically) so the diff shows relevant changes?

Reply:
Whoops, that was unintentional!  I'll push that...